src/data/propertyPaths.ts

```diff
@@ -1064,7 +1100,9 @@ export const parsedPropertyPaths: Record<string, string[]> = {
     'status',
     'subject',
     'derivedFrom',
-    'timing[x]',
+    'timingTiming',
+    'timingPeriod',
+    'timingdateTime',
```

Comment:
Ugh this is super annoying but there are a few of these codeable concepts that I think need to be capitalized (I feel like this should be fixed in the Structure Defs but whatever. Ex: `timingdateTime` here should be `timingDateTime` I believe: https://hl7.org/fhir/R4/deviceusestatement.html. I think you can make this a simple fix my just capitalizing the first one before concatenating it with timing.